src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3042,56 +3042,85 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _core.ClearHoveredCell();
     }
 
-    winrt::fire_and_forget TermControl::_hoveredHyperlinkChanged(IInspectable /*sender*/,
-                                                                 IInspectable /*args*/)
+    // Attackers abuse Unicode characters that happen to look similar to ASCII characters. Cyrillic for instance has

```

Comment:
[You should suppress whitespace changes!](https://github.com/microsoft/terminal/pull/15488/files?w=1)